### PR TITLE
Updated RxSwift to 6.0 and ios.deployment_target

### PR DESCRIPTION
**Pull request by Loupehope**
Updated:
- RxSwift to 6.0
- ios.deployment_target to 9.0
- Update code with new RxSwift Features

### CODE REVIEW

**Comment by freak4pc** [comment]
@jdisho you're listed as the current owner on the project, care to review and cut a new release? 

Thanks @Loupehope !

**Review by jdisho — CHANGES_REQUESTED** [review]
Thanks for your work @Loupehope 😊

However, this PR contains work that IMO it **should** be addressed in separate PRs.  I would appreciate if you can leave only the work about RxSwift 6.0 here and open new PRs about the code indentation part and the #98.

**Comment by Loupehope** [comment]
@jdisho thanks for review!

I returned the changes related to indentation and #98.

File: Sources/RxKeyboard/RxKeyboard.swift [hunk 1]
```diff
@@ -50,17 +50,10 @@ public class RxKeyboard: NSObject, RxKeyboardType {
   // MARK: Initializing
 
   override init() {
-    #if swift(>=4.2)
-      let keyboardWillChangeFrame = UIResponder.keyboardWillChangeFrameNotification
-      let keyboardWillHide = UIResponder.keyboardWillHideNotification
-      let keyboardFrameEndKey = UIResponder.keyboardFrameEndUserInfoKey
-      let applicationDidFinishLaunching = UIApplication.didFinishLaunchingNotification
-    #else
```
Comment (jdisho) [inline Sources/RxKeyboard/RxKeyboard.swift:58]: Is this change intentional? Why should we drop support for versions lower than 4.2?

**Comment by jdisho** [comment]
The CI keeps failing. Mind having a look? 😊

File: Sources/RxKeyboard/RxKeyboard.swift (hunk 1 above)
Comment (Loupehope) [reply Sources/RxKeyboard/RxKeyboard.swift:58]: In .podspec and package.swift files swift 5.0 is listed as minimal supported version, isn't it?

File: RxKeyboard.podspec [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
   s.homepage         = 'https://github.com/RxSwiftCommunity/RxKeyboard'
   s.license          = { :type => 'MIT', :file => 'LICENSE' }
   s.author           = { 'Suyeol Jeon' => 'devxoul@gmail.com' }
   s.source           = { :git => 'https://github.com/RxSwiftCommunity/RxKeyboard.git',
                          :tag => s.version.to_s }
   s.source_files     = 'Sources/**/*.swift'
-  s.frameworks       = 'UIKit', 'Foundation'
+  s.frameworks       = 'UIKit'
   s.requires_arc     = true
   s.swift_version    = "5.0"
```
Comment (jdisho) [inline RxKeyboard.podspec:13]: Since RxSwift's Swift version is `5.1`, I think this needs to reflected too.

**Comment by jdisho** [comment]
We would appreciate your help @freak4pc regarding the CI 😊

**Comment by Loupehope** [comment]
> We would appreciate your help @freak4pc regarding the CI

@jdisho I fixed the travis.yml file and increased the swift version to 5.1. It looks like all the checks passed!

**Review by jdisho — APPROVED** [review]
Awesome! Thank you
